sbnana/CAFAna/Core/Var.h

```diff
@@ -76,6 +76,9 @@ namespace ana
   /// \brief Equivalent of \ref Var acting on \ref caf::SRSpill
   typedef _Var<caf::SRSpillProxy> SpillVar;
 
+  /// \brief Equivalent of \ref Var acting on \ref caf::SRTrueInteraction
+  typedef _Var<caf::SRTrueInteractionProxy> TruthVar;
```

Comment:
Line 30 above can add to the comment that it's a template for `TruthVar` as well.